src/extensions/styles/migrators/IBLabelToIDMigrator.js

```
@@ -8,30 +8,43 @@ import { getIBDataItem } from './utils';
+	// Use for...of for better performance
+	for (const relation of relations) {
+		if (!isRelationEligible(relation)) continue;
 
 		const dataItem = getIBDataItem(relation);
+		if (!dataItem?.sid) continue;
 
-		if (!dataItem || !dataItem?.sid) return;
-
+		// Direct property mutations for better performance
 		relation.sid = dataItem.sid;
 		delete relation.settings;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)